src/components/views/dialogs/DevtoolsDialog.tsx

```diff
@@ -119,11 +120,15 @@ const DevtoolsDialog: React.FC<IProps> = ({ roomId, onFinished }) => {
                 {(cli) => (
                     <>
                         <div className="mx_DevTools_label_left">{label}</div>
-                        <div className="mx_DevTools_label_right">{_t("Room ID: %(roomId)s", { roomId })}</div>
+                        <CopyableText className="mx_DevTools_label_right" getTextToCopy={() => roomId} border={false}>
+                            {_t("Room ID: %(roomId)s", { roomId })}
+                        </CopyableText>
                         <div className="mx_DevTools_label_bottom" />
-                        <DevtoolsContext.Provider value={{ room: cli.getRoom(roomId) }}>
-                            {body}
-                        </DevtoolsContext.Provider>
+                        {cli.getRoom(roomId) && (
+                            <DevtoolsContext.Provider value={{ room: cli.getRoom(roomId)! }}>
+                                {body}
+                            </DevtoolsContext.Provider>
+                        )}
```

Comment:
this change looks unrelated to the PR?

Reply:
Very true, it was there to appease the TypeStrict checks